src/app/containers/StoryPromo/IndexAlsos/index.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useContext } from 'react';
+import { oneOfType, arrayOf, shape, number, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import pathOr from 'ramda/src/pathOr';
+import MediaIndicator from '@bbc/psammead-media-indicator';
+import {
+  IndexAlsos,
+  IndexAlso,
+  IndexAlsosUl,
+  IndexAlsosLi,
+} from '@bbc/psammead-story-promo/index-alsos';
+import { ServiceContext } from '../../../contexts/ServiceContext';
+
+const IndexAlsosContainer = ({ alsoItems, script, service }) => {
+  const {
+    translations: { media: mediaTranslations },
+  } = useContext(ServiceContext);
+
+  const getMediaType = (cpsType, mediaType) => {
+    const isPGL = cpsType === 'PGL';
+    const isMedia = cpsType === 'MAP';
+    const media = mediaType || 'Video';
+
+    if (!isPGL && !isMedia) {
+      return null;
+    }
+
+    const type = isPGL ? 'photogallery' : media.toLowerCase();
+
+    return type;
+  };
+
+  const IndexAlsosMediaIndicator = (cpsType, mediaType) => {
+    const indexAlsosMediaType = getMediaType(cpsType, mediaType);
+
+    return (
+      indexAlsosMediaType && (
+        <MediaIndicator
+          service={service}
+          type={indexAlsosMediaType}
+          indexAlsos
+        />
+      )
+    );
+  };
+
+  return (
+    <IndexAlsos offScreenText="Related content">
+      {alsoItems.length > 1 ? (
```

Comment:
This is alot of duplicated code for `< 1` & `> 1`. Could you maybe do the same logic, but do like
```
const IndexAlsosUlComponent = alsoItems.length > 1 ? IndexAlsosUl : Fragment;
const IndexAlsoComponent = alsoItems.length > 1 ? IndexAlsosLi : IndexAlso;
```
And loop over them the exact same way with `{alsoItems.slice(0, 3).map(item => {`